python/lsst/ts/scheduler/driver.py

```diff
@@ -15,8 +15,6 @@
 from lsst.ts.scheduler.setup import EXTENSIVE, WORDY
 from lsst.ts.scheduler.kernel import read_conf_file
 from lsst.ts.scheduler.kernel import Field, SurveyTopology
-from lsst.ts.scheduler.proposals import ScriptedProposal
-from lsst.ts.scheduler.proposals import AreaDistributionProposal, TimeDistributionProposal
 from lsst.ts.scheduler.fields import FieldsDatabase
 from lsst.ts.scheduler.lookahead import Lookahead
```

Comment:
I think Lookahead can also be removed as this is for the Proposal Based Scheduler.